aredis/connection.py

```diff
@@ -367,7 +367,7 @@ class BaseConnection:
     def __init__(self, retry_on_timeout=False, stream_timeout=None,
                  parser_class=DefaultParser, reader_read_size=65535,
                  encoding='utf-8', decode_responses=False,
-                 *, loop=None):
+                 *, loop=None, client_name=None):
```

Comment:
IMO `client_name` should be defined before `loop` as it is application configuration unlike `loop`, but this is very subjective.

Reply:
Changed.